bitnami/prometheus-operator/templates/prometheus/thanos-service.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.thanos.create }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "prometheus-operator.prometheus.fullname" . }}-thanos
+  labels: {{- include "prometheus-operator.prometheus.labels" . | nindent 4 }}
```

Comment:
should we add a `app.kubernetes.io/component: thanos` label?

question: would it have made sense if this service config was included in the existing `templates/prometheus/service.yaml`

Reply:
> should we add a app.kubernetes.io/component: thanos label?

Yes, why not?

> question: would it have made sense if this service config was included in the existing templates/prometheus/service.yaml

We could have done it that way but I think it's less confusing to have a specific service to access Thanos